packages/cdktf-cli/src/bin/cmds/handlers.ts

```diff
@@ -14,6 +14,7 @@ import {
   TerraformModuleConstraint,
   TerraformProviderConstraint,
 } from "@cdktf/provider-generator";
+import Table from "cli-table";
```

Comment:
I have no opinion on this yet – I'd like to get rid of React (as it seems to complicate things when working with hooks, etc.) but I suspect that building an interactive UI without it might be harder – so that would be my only reason to keep `ink`. Ah, well, error handling is also something were we make use of `ink` features iirc.

Reply:
Ah, good one. I haven't used ink before, but I'll try to take a closer look at it.

Reply:
Here is an ink table component: https://github.com/maticzav/ink-table